### PR TITLE
fix(component): mega menu mobile viewport checker

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes https://github.com/unimelb/pattern-lib/issues/1564

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] Make sure to do not repeat yourself (DRY)
- [x] Snapshots tested
- [x] A11y tested
- [x] CSS utilises BEM naming convention and structure
- [x] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [x] Crossbrowser testing (IE11, Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [x] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [ ] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [ ] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [ ] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 9b761ffe54aad7b6ef5e3ef4a2294fbd7289f378

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/60627ed887d08f000776d070

File: components/megamenu/MegaMenu.vue [hunk 1]
```diff
@@ -190,6 +190,7 @@ import MegaMenuTopNavigation from './MegaMenuTopNavigation.vue';
 import Logo from '../logo/Logo.vue';
 import { WIDTH_900 } from '../../helpers/viewports';
```
Comment (jaredh-um) [inline components/megamenu/MegaMenu.vue:191]: can this line be removed now?

Comment (gepifanio) [reply components/megamenu/MegaMenu.vue:191]: Not really, as we still check the screen width to the menu items, be opened in "mobile" or "desktop" view if there is a desktop with a small size, still should be able to open the menu items like a "mobile"

Comment (jaredh-um) [reply components/megamenu/MegaMenu.vue:191]: Sure. I'm not really a big fan of arbitrary numbers like "900" though; what's that been based on? Nothing from a design perspective. We don't need to change anything here now, but I feel this kind of number should relate to a certain breakpoint or media query, not some random fixed number.

**Review by jaredh-um — APPROVED** [review]

File: components/megamenu/MegaMenu.vue (hunk 1 above)
Comment (gepifanio) [reply components/megamenu/MegaMenu.vue:191]: Yes I agree, we need to improve this sort of things

**Pull request merged**